src/select.vue

```diff
@@ -35,11 +35,8 @@ export default {
     }
   },
   methods: {
-    selectValue (option) {
-      this.value = option.value
-      this.name = option.name
-      let event = new Event('change')
-      this.$el.dispatchEvent(event)
+    selectValue ({value}) {
+      this.$emit('input', value)
```

Comment:
Having the native `change` event is still necessary. But I think I'll update this myself because it wasn't really useful without any data